### ISSUE TITLE
A bug appear in samples in notebooks

### ISSUE THREAD

**Issue by Bigwode** [albu/albumentations#206]
## To Reproduce

1.run migrating_from_torchvision_to_albumentations.ipynb

2. 
from torch.utils.data import DataLoader
trainloader = DataLoader(torchvision_dataset, batch_size=1,
                                          shuffle=True, num_workers=1)
for i, (image, label) in enumerate(trainloader):
    print(i, (image.shape, label))
add these lines of code,
and we can find that image.shape = torch.Size([1, 224, 224, 3]), but pytorch needs the inputs.shape=[1, 3,224, 224]

## Environment

 - Albumentations version (e.g., 0.1.8):
 - Python version (e.g., 3.7):
 - OS (e.g., Linux):
 - How you installed albumentations (`conda`, `pip`, source):
 - Any other relevant information:

**Comment by ternaus** [comment]
You need to transform from numpy array to the torch tensor. The way to do it is:
```
from albumentations.pytorch.functional import img_to_tensor
```

and within a DataLoader to cast `img_to_tensor` operation to the augmented image.

```
...

return img_to_tensor(image), label
```

I updated the notebook in https://github.com/albu/albumentations/pull/208
